lib/mayaUsd/commands/baseListJobContextsCommand.cpp

```diff
@@ -48,9 +49,30 @@ _GetInfo(const MArgDatabase& argData, const char* optionName)
     return emptyInfo;
 }
 
+MString convertDictionaryToText(const VtDictionary& settings)
```

Comment:
This code was simply moved from within the functions below, which had multiple copies of it.